src/quo2/components/settings/data_item/view.cljs

```diff
@@ -118,13 +118,13 @@
          {:accessibility-label :data-item
           :disabled            (not icon-right?)
           :on-press            on-press
-          :style               (merge (style/container size card? blur? theme) container-style)}
+          :style               (merge container-style (style/container size card? blur? theme))}
```

Comment:
I think we should swap this order so container-style prop is an overwrite and gets priority

Reply:
I believe it's not a good idea here as the component's core style is from the `container` style function. If any styles got overwritten by `container-style` props, it would affect/break the component style. 

We use the `container-style` prop mainly for adding the padding or margin (if the component uses padding inside it) from the screen. If there are any style changes (on the component type used in the screen) apart from it, this would help us to identify and raise them to the design team.

Reply:
true it should not really affect the usability of this feature this way. I guess the ideal way is that devs don't hack other styles (non margin, z-index etc) into the container-styles.
anyway as long as this works it should be fine, can always be adjusted later if it is needed